build.sh

```diff
@@ -30,6 +30,12 @@ if [ $? -ne 0 ]; then
   exit 1
 fi
 
+if [ "$OAUTH_AGENT" == 'FINANCIAL' ]; then
```

Comment:
The whole setup got a bit complicated in my opinion. E.g., these config files are actually only used in the development mode. The spa-deployments scripts override them. This makes it a bit hard sometimes to understand where some settings come from in the end. Not sure if we should spend time now to change this...